edweiss-app/__test__/app/login/index.test.tsx

```diff
@@ -0,0 +1,217 @@
+/**
+ * @file index.test.tsx
+ * @description Test suite for /login/index screen and LoadingPageComponent
+ * @author Adamm Alaoui
+ */
+
+// ------------------------------------------------------------
+// --------------- Import Modules & Components ----------------
+// ------------------------------------------------------------
+
+import Login, { LoadingPageCompoment } from '@/app/login';
+import { callFunction, signInAnonymously, signInWithGoogle } from '@/config/firebase';
+import { fireEvent, render, waitFor } from '@testing-library/react-native';
+import { router } from 'expo-router';
+import React from 'react';
+
+// ------------------------------------------------------------
+// -----------------  Mocking dependencies    -----------------
+// ------------------------------------------------------------
```

Comment:
Oh that's odd, why would we loose point for comments ? More precisely, it's a section definition, so that when you scroll on the code it easier to find our way in it. Don't you think so.
Personally, it safe to keep it.  Can we keep it ?